VmAgent.Core/VmAgent.Core.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <PackageVersion>1.1.8</PackageVersion>
+    <PackageVersion>1.1.9</PackageVersion>
```

Comment:
do we need to increase the version number here?

Reply:
So this packages AgentInterfaces along with VMCore, VM Agent only consumes VmCore, not AgentInterfaces separately. Whereas CP only consumes AI. Thus a change in AI needs a core version bump :)